xml/System.Text.Json/JsonElement+ArrayEnumerator.xml

```diff
@@ -172,9 +172,17 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an enumerator that iterates through a collection.</summary>
+        <returns>An enumerator for <see cref="T:System.Text.Json.JsonElement" /> objects that can be used to iterate through the collection.</returns>
```

Comment:
Maybe we should avoid using "objects" here, since it is an overloaded term, JSON object or .NET object and this is meant for "JSON arrays".

```suggestion
        <returns>An enumerator for an array of <see cref="T:System.Text.Json.JsonElement" /> that can be used to iterate through the collection.</returns>
```